### PR TITLE
sig-gov de-dupping evolution continues #4781

### LINKED ISSUE #4781: sig-gov de-dupping evolution continues..
we merged and want to iterate again now on [sig-governance.md](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md) based on some comments that were left from folks and what we see as gaps now that we've collated 

tasks:
- [ ] take the first section about general sig operations and simplify this so it isn't a repeat of the chair duties below. a way to solve this could be listing out the general activity with a link to it's canonical source without the detail 
example:
    - [community meeting](link to the procedure around community meetings)
- [ ] create sections for the duties in the chair section to break up the text, ie: "community updates:", "operations:", "cncf/kubecon stuff" as loose examples
- [x] whitespacing issues
- [x] line 69-72 needs editorial changes
- [x] general formatting and other style guide changes

### PR DESCRIPTION
**Pull request by divyabhushan**
**Which issue(s) this PR fixes**:
Ref #4781

File modified: [sig-governance.md]
Enhancements/editions:

[x] white spacing issues
[x] line 69-72 needs editorial changes
[x] general formatting and other style guide changes

@parispittman 

[sig-governance.md]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md

### CODE REVIEW

File: committee-steering/governance/sig-governance.md [hunk 1]
```diff
@@ -57,32 +57,35 @@ Within this section "Lead" refers to someone who is a member of the union
...
+  between Chairs, Tech Leads and Subproject Owners.
 
 #### Requirements
+
 - Leads *MUST* be at least a ["member" on our contributor ladder] to
 be eligible to hold a leadership role within a SIG.
 - SIGs *MAY* prefer various levels of domain knowledge depending on the
 role. This should be documented.  
-- People management interests - there's a lot of us!    
+- People management interests - there's a lot of us!<!-- removed trailing spaces -->
```
[the remark this answers is not in the document]
Comment (divyabhushan) [reply committee-steering/governance/sig-governance.md]: Thanks @mrbobbytables for the review. 
I have removed all the comments.

**Comment by divyabhushan** [comment]
> Thanks for working on this 👍 A few small suggestions.
@mrbobbytables
I have removed all the comments from the doc.
I hope the rest is ok.
Thanks.

File: committee-steering/governance/sig-governance.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # SIG Roles and Organizational Governance
 
-This charter adheres to the conventions described in the [Kubernetes Charter README].
+This charter adheres to the conventions described in the [Kubernetes Charter README].\
```
Comment (mrbobbytables) [inline committee-steering/governance/sig-governance.md:3]:
```suggestion
This charter adheres to the conventions described in the [Kubernetes Charter README].
```
Removed `\`

File: committee-steering/governance/sig-governance.md [hunk 3]
```diff
@@ -256,3 +254,5 @@ Issues impacting multiple subprojects in the SIG should be resolved by either:
 [contributor guide]: /contributors/guide/README.md
 [devel]: /contributors/devel/README.md
 [#tech-lead]: #Tech-Lead
+[Google group]: https://groups.google.com/forum/#!forum/kubernetes-sig-config
```
[the remark this answers is not in the document]
Comment (divyabhushan) [reply committee-steering/governance/sig-governance.md:257]: ok, noted.

**Comment by divyabhushan** [comment]
@mrbobbytables 
Applied the changes, kindly review.
Thanks.

**Comment by parispittman** [comment]
thank you so much for this and your patience as I worked through a new employer :)

/lgtm
/approve

**Comment by divyabhushan** [comment]
> thank you so much for this and your patience as I worked through a new employer :)
> 
> /lgtm
> /approve

Thank you @parispittman and @mrbobbytables to help me close my first PR :)

**Pull request merged**